analyzer/tests/functional/analyze_and_parse/test_files/multi_error_skipped.output

```diff
@@ -6,7 +6,7 @@ CHECK#CodeChecker check --build "make multi_error simple1" --output $OUTPUT$ --q
 [] - Starting build ...
 [] - Build finished successfully.
 [] - Starting static analysis ...
-[] - [1/2] clangsa analyzed simple1.cpp successfully.
+[] - [1/1] clangsa analyzed simple1.cpp successfully.
```

Comment:
Shouldn't be the "Skipped compilation commands:..." message appear in the output here?